sql/plan/insert.go

```diff
@@ -164,18 +164,7 @@ func (ii *InsertInto) WithChildren(children ...sql.Node) (sql.Node, error) {
 
 // CheckPrivileges implements the interface sql.Node.
 func (ii *InsertInto) CheckPrivileges(ctx *sql.Context, opChecker sql.PrivilegedOperationChecker) bool {
```

Comment:
ditto zach's comment about deleting this interface

Reply:
we doing this or as a follow up?

Reply:
It'll be this PR